### PR TITLE
Rewrite and fix runtimes.bref.sh

### PR DESCRIPTION
**Pull request by sgrodzicki**
Fixes https://github.com/brefphp/bref/issues/537

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {
     "require": {
         "bref/bref": "0.4.1",
-        "aws/aws-sdk-php": "^3.85",
         "slim/slim": "^3.11",
         "slim/twig-view": "^2.4",
-        "roave/security-advisories": "dev-master"
+        "roave/security-advisories": "dev-master",
```
Comment (mnapoli) [inline composer.json]: 👍

File: index.php [hunk 2]
```diff
@@ -55,54 +64,72 @@
...
+    $url = 'https://api.github.com/repos/brefphp/bref/releases';
+
+    $response = $client->get($url);
+    $json = $response->getBody()->getContents();
+    $releases = json_decode($json, true);
+
+    $versions = [];
+
+    foreach ($releases as $release) {
+        // Skip prereleases (e.g., 0.5.14-beta1)
```
Comment (mnapoli) [inline index.php]: 👌 good call!

File: index.php [hunk 3]
```diff
@@ -55,54 +64,72 @@
...
         ];
     }
 
     return $layers;
 }
+
+function listVersions(): array
+{
+    $client = new Client();
+    $url = 'https://api.github.com/repos/brefphp/bref/releases';
```
Comment (mnapoli) [inline index.php]: I wonder if we risk hitting the API limit 🤔 

I guess we'll see, we can always add a basic cache later I think (e.g. cache in `/tmp` or in apcu).

**Comment by sgrodzicki** [comment]
> One detail, if you have time to fix the `/embedded` URL as well: this is the embedded version that is included in this page: https://bref.sh/docs/runtimes/#layer-version-

The `/embedded` URL should work just fine. I'm assuming the latest Bref version for that page, so no changes in the template are needed.

File: index.php (hunk 3 above)
Comment (sgrodzicki) [reply index.php]: We'll probably hit the API limit quickly (I've even experienced it while working on that PR). I'll add some caching.

Comment (mnapoli) [reply index.php]: 👍

**Comment by mnapoli** [comment]
Right, I didn't think about the latest version! Makes sense!

File: index.php [hunk 4]
```diff
@@ -78,12 +80,21 @@
 
 function listLayers(string $version, string $region): array
 {
-    $client = new Client();
-    $url = 'https://raw.githubusercontent.com/brefphp/bref/' . $version . '/layers.json';
+    $cache = new FilesystemAdapter();
```
Comment (mnapoli) [inline index.php]: Where are the files stored?

It should be in `/tmp`, everything else is readonly on AWS Lambda.

Comment (sgrodzicki) [reply index.php]: It will use `sys_get_temp_dir()` to determine the proper location. Will that work on AWS Lambda or should I make it explicit?

Comment (mnapoli) [reply index.php]: You know what let's try it out ^^

**Comment by mnapoli** [comment]
This is PERFECT! Everything works:

- https://runtimes.bref.sh/
- https://bref.sh/docs/runtimes/#layer-version-

Thank you!

**Comment by Nyholm** [comment]
Wow, awesome!

**Pull request merged**